src/models/item_sets.py

```
@@ -0,0 +1,45 @@
+from dataclasses import dataclass, field
+from typing import Dict, List
+from uuid import uuid4
+from .base_types import ItemRarity
+from .effects.base import BaseEffect
+from .effects.item_effects import LifestealEffect, OnHitEffect, StatModifierEffect
```

Ruff (0.8.2)

`.effects.item_effects.LifestealEffect` imported but unused

Remove unused import

(F401)